pom.xml

```diff
@@ -282,11 +280,6 @@
         <artifactId>commons-logging</artifactId>
         <version>1.1.1</version>
       </dependency>
-      <dependency>
-        <groupId>info.ganglia.gmetric4j</groupId>
-        <artifactId>gmetric4j</artifactId>
-        <version>${gmetric4j.version}</version>
-      </dependency>
```

Comment:
Is this dependency not used in the codebase?

Reply:
yes.